Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -1238,6 +1238,26 @@ UpdateFspConfig (
   if (FeaturePcdGet (PcdEnablePciePm)) {
     StoreRpConfig (FspsConfig);
   }
+
+#if FixedPcdGetBool(PcdFusaSupport) == 1
+    FspsConfig->FusaConfigEnable = SiCfgData->FusaConfigEnable;
```

Comment:
"== 1" could be removed.
To avoid potential KW issue, I think it need check if SiCfgData is NULL as below.
if (SiCfgData != NULL) {

Reply:
Should be fixed now.